addons/microsoft_calendar/models/calendar.py

```diff
@@ -103,8 +114,34 @@ def _is_matching_timeslot(self, start, stop, allday):
 
         return (event_start, event_stop) == (start, stop)
 
+    def _forbid_recurrence_update(self):
+        """
+        Suggest user to update recurrences in Outlook due to the Outlook Calendar spam limitation.
+        """
+        error_msg = _("Due to an Outlook Calendar limitation, recurrence updates must be done directly in Outlook.")
```

Comment:
thse strings needs to be added to microsoft_calendar.pot
Generate the pot file with
./odoo-bin  -d calendar --addons-path=addons,../enterprise  -l en --i18n-export=/tmp/prout.pot --modules=microsoft_calendar
Then you compare /tmp/prout.pot  and microsoft_calendar.ot and add to your commit the relevant parts.